lib/workers/api_use.js

```diff
@@ -66,70 +66,88 @@ function postProcess(parsedFiles, filenames, preProcess, packageInfos, source, t
 
     parsedFiles.forEach(function(parsedFile, parsedFileIndex) {
         parsedFile.forEach(function(block) {
-            if ( ! block.local[target])
-                return;
-
-            block.local[target].forEach(function(definition) {
-                var name = definition.name;
-                var version = block.version || packageInfos.defaultVersion;
-
-                if ( ! preProcess[source] || ! preProcess[source][name]) {
-                    throw new WorkerError('Referenced groupname does not exist / it is not defined with @apiDefine.',
+            var loopCounter = 0;     //add a loop counter to have a break condition when the recursion depth exceed a predifined limit
+            while (block.local[target]) {
+                if (loopCounter > 10) {
+                    throw new WorkerError('recursion depth exceeds limit with @apiUse',
                         filenames[parsedFileIndex],
                         block.index,
                         messages.common.element,
                         messages.common.usage,
                         messages.common.example,
                         [
-                            { 'Groupname': name }
+                           { 'Groupname': block.name }
                         ]
                     );
                 }
+                
```

Comment:
this line has whitespaces (we should really have an automatic tool to look/fix these minor issues...)